library/swift/src/filters/ResponseFilter.swift

```diff
@@ -1,7 +1,28 @@
+@_implementationOnly import EnvoyEngine
 import Foundation
 
+extension EnvoyHTTPFilter {
+  /// Initializer
```

Comment:
I'd either remove this or provide an actual description

Reply:
Sure, I'll elaborate.